conda_forge_tick/audit.py

```diff
@@ -123,3 +124,48 @@ def main(args):
                 finally:
                     with open(f"audits/{k}/{node}_{version}.{ext}", "w") as f:
                         v["writer"](deps, f)
+
+    grayskull_files = os.listdir("audits/grayskull")
+    bad_inspections = {}
+    if "_net_audit.json" in grayskull_files:
+        grayskull_files.pop(grayskull_files.index("_net_audit.json"))
+        with open("audits/grayskull/_net_audit.json", "w") as f:
+            bad_inspections = load(f)
+    for node, attrs in gx.nodes("payload"):
+        if not attrs.get("version"):
+            continue
+        node_version = f"{node}_{attrs['version']}"
+        if node_version in bad_inspections:
+            continue
+        # construct the expected filename
+        expected_filename = f"{node_version}.yml"
+        if expected_filename in grayskull_files:
+            # load the feedstock with the grayskull meta_yaml
+            try:
+                with open(
+                    os.path.join("audits/grayskull", expected_filename), "r",
+                ) as f:
+                    new_attrs = load_feedstock(node, {}, meta_yaml=f.read())
```

Comment:
Why reload here? This pulls from the internet. Maybe use what is in the serialized copy of the graph?

Reply:
This is loading up the grayskull version of the feedstock

Reply:
Ah. I'd expect this block of code to take of order 20 minutes to run just itself then. It is not threaded like we did for making the graph, right?

Reply:
It should be a lot shorter after the initial burn in, since we don't rerun things we've already done, but I'll put it on a thread just for the principle.